src/com/hms/flexyosisoftconnector/OSIsoftServer.java

```diff
@@ -299,6 +300,37 @@ public int initTags() throws JSONException {
           if (res != NO_ERROR) retval = res;
         }
         break;
+        // OMF OCS setup
+      case OSIsoftConfig.ocs:
+          Logger.LOG_SERIOUS("in ocs logic");

```

Comment:
This log output should be more formal

Reply:
forgot to remove this. it was to check i put the write thing on that config file